swagger.json

```diff
@@ -3105,6 +3168,76 @@
                 }
             }
         },
+        "VehicleStats": {
+            "type": "object",
+            "description": "Engine state and aux input data for a vehicle.",
+            "properties": {
+                "engineState": {
+                    "$ref": "#/definitions/ObjectStatEngineState"                    
+                },
+                "auxInput1": {
+                    "type": "object",
+                    "required": ["name", "values"],
+                    "properties": {
+                        "name": {
+                            "type": "string",
+                            "example": "Boom"
```

Comment:
Could we enumerate the DIGIO input types here? https://swagger.io/docs/specification/2-0/enums/